atest/robot/running/skip_with_template.robot

```diff
@@ -0,0 +1,22 @@
+*** Settings ***
+Suite Setup     Run Tests  --skip skip-this --SkipOnFailure skip-on-failure    running/skip/
```

Comment:
Both `--skip` and `--SkipOnFailure` are unnecessary as none of the executed tests have these tags. All options can be replaced with `${EMPTY}`.